src/Profile.php

```diff
@@ -778,6 +778,334 @@ public function showForm($ID, array $options = [])
+                'label' => null,
+                'rights' => null,
+                'scope' => 'entity'
+            ], $options);
+
+            return [
+                'rights' => $options['rights'] ?? Profile::getRightsFor($itemtype, $interface),
+                'label'  => $options['label'] ?? $itemtype::getTypeName(Session::getPluralNumber()),
+                'field'  => $options['field'] ?? $itemtype::$rightname,
+                'scope' => $options['scope']
+            ];
+        };
+
+        static $all_rights = null;
+
+        if ($all_rights === null) {
+            $dropdown_rights = (new Profile())->getRights();
+            unset($dropdown_rights[DELETE]);
+            unset($dropdown_rights[UNLOCK]);
+
+            $all_rights = [
+                'central' => [
+                    'tracking' => [
+                        'itilobjects' => [
+                            $fn_get_rights(TicketTemplate::class, 'central', [
+                                'label' => _n('Template', 'Templates', Session::getPluralNumber())
+                            ]),
+                            $fn_get_rights(PendingReason::class, 'central'),
+                        ],
+                        'tickets' => [
+                            $fn_get_rights(Ticket::class, 'central'),
+                            $fn_get_rights(TicketCost::class, 'central'),
+                            $fn_get_rights(TicketRecurrent::class, 'central'),
+                        ],
+                        'followups_tasks' => [
+                            $fn_get_rights(ITILFollowup::class, 'central'),
+                            $fn_get_rights(TicketTask::class, 'central'),
+                        ],
+                        'validations' => [
+                            $fn_get_rights(TicketValidation::class, 'central'),
+                        ],
+                        'visibility' => [
+                            $fn_get_rights(Stat::class, 'central'),
+                            $fn_get_rights(Planning::class, 'central'),
+                        ],
+                        'planning' => [
+                            $fn_get_rights(PlanningExternalEvent::class, 'central'),
+                        ],
+                        'problems' => [
+                            $fn_get_rights(Problem::class, 'central'),
+                        ],
+                        'changes' => [
+                            $fn_get_rights(Change::class, 'central'),
+                            $fn_get_rights(ChangeValidation::class, 'central'),
+                            $fn_get_rights(RecurrentChange::class, 'central'),
+                        ],
+                    ],
+                    'tools' => [
+                        'general' => [
+                            $fn_get_rights(Reminder::class, 'central'),
+                            $fn_get_rights(RSSFeed::class, 'central'),
+                            $fn_get_rights(SavedSearch::class, 'central'),
```

Comment:
Should we add `Public` term? @orthagh 

Before:
![image](https://github.com/glpi-project/glpi/assets/33253653/13801346-faa4-474b-b80a-8d1517e00136)

After:
![image](https://github.com/glpi-project/glpi/assets/33253653/1607556a-22fa-4692-a780-8ae8d29449e7)